src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -65,4 +69,21 @@ void register_nullLogin_notOk() {
                 () -> registrationService.register(registrateFirstUser));
     }
 
+    @Test
+    void register_userSuccessRegistration_Ok() {
+        registrateNotSameFirstLoginUser.setLogin("nafaniaNotSame");
+        registrateNotSameFirstLoginUser.setPassword("youngBob");
+        registrateNotSameFirstLoginUser.setAge(20);
+        storageDao.add(registrateFirstUser);
+        storageDao.add(registrateNotSameFirstLoginUser);

```

Comment:
It seems like there should be a register method here

Reply:
i hope, finally, i understand you